deploy/helms/legion-core/templates/metrics/grafana.yaml

```diff
@@ -34,6 +34,23 @@ spec:
         {{- else }}
           value: "http://{{ .Release.Name }}-graphite.{{ .Release.Namespace }}.svc.{{ .Values.localDomain }}:80"
         {{- end }}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /login
+            port: 3000
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        readinessProbe:
```

Comment:
Do we really need liveness and readiness probe in this case?

Reply:
addressed